client/infraAccBndlGrp_service.go

```diff
@@ -413,8 +413,8 @@ func (sm *ServiceManager) ReadRelationinfraRsNetflowMonitorPolFromPCVPCInterface
 
 	for _, contItem := range contList {
 		paramMap := make(map[string]string)
-		paramMap["tnNetflowMonitorPolName"] = models.G(contItem, "tDn")
-		paramMap["fltType"] = models.G(contItem, "fltType")
+		paramMap["target_dn"] = models.G(contItem, "tDn")
+		paramMap["flt_type"] = models.G(contItem, "fltType")
```

Comment:
I don't think we use the snake case for this method but would more use the same case as the APIC model like tDn and fltType. Did you check in other files in go client?